specification/web/resource-manager/Microsoft.Web/stable/2018-02-01/WebApps.json

```diff
@@ -12976,7 +12994,7 @@
         ],
         "summary": "Gets a named operation for a network trace capturing (or deployment slot, if specified).",
         "description": "Gets a named operation for a network trace capturing (or deployment slot, if specified).",
-        "operationId": "WebApps_GetNetworkTracesSlotV2",
+        "operationId": "WebApps_GetNetworkTracesV2Slot",
```

Comment:
This is a breaking change that will require a major version bump in any published SDKs - are you sure this is necessary?

Reply:
The bug was merged last week, so if no SDKs were published since then, we are fine. Do you know when was the last published date?

Reply:
Reverted the change.